db/migrate/20200505013836_add_avatar_photo_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvatarPhotoToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :avatar_photo, :string
+  end
+end
```

Comment:
Do we need this migration at all if we can access the avatar photo through the uploader?